app/routes/auth.py

```
@@ -53,55 +37,16 @@ def hash_password(password):
-    return False
-
-def hash_refresh_token(token: str) -> str:
-    return hashlib.sha256(token.encode("utf-8")).hexdigest()
-
-def persist_refresh_token(db: Session, user_id: int, token: str, expires_at: datetime):
-    token_hash = hash_refresh_token(token)
-    db_token = RefreshToken(user_id=user_id, token_hash=token_hash, expires_at=expires_at, revoked=False)
-    db.add(db_token)
-    db.commit()
-    return db_token
+    expire = datetime.utcnow() + timedelta(minutes=ACCESS_TOKEN_EXPIRE_MINUTES)
```

Ruff (0.15.6)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)